src/main/webapp/destinationCreationSuccess.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Scavenger Hunt</title>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500&amp;display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="common-style.css" />
+    <link rel="stylesheet" href="success.css" />
+    <script src="success.js"></script>
+  </head>
+  <body class="background" onload="confirmCreation()">
+    <div class="content">
+      <div id="confirmation-block">
+        <h1>Destination Successfully Created!</h1>
+      </div>
+      <div class="button-row">
+        <a href="index.html" class="button">Home Page</a>
+        <a href="generateHunt.html" class="button">Generate a Hunt</a>
```

Comment:
maybe you should have another button to add another destination?

Reply:
Yea that's not a bad idea